src/Makefile

```diff
@@ -507,7 +507,7 @@ inclean:
 	$(RM) $(INFILES)
 headersclean:
 	$(RM) $(HEADERS)
-clean: docclean pythonclean cscopeclean headersclean
+clean: docclean pythonclean cscopeclean inclean headersclean
 distclean: inclean
```

Comment:
I believe using 'inclean' in the clean target is wrong, it belong to distclean as it remove files generated by configure.

Reply:
Do we lack a dependency of distclean on clean?